apps/web/src/App/Router/AppRouter.tsx

```diff
@@ -92,11 +93,10 @@ function AppRouter(): ReactElement {
                           element={<Overview></Overview>}
                         ></Route>
                         <Route path="/stake" element={<Stake></Stake>}></Route>
-                        {/*
                         <Route
                           path="/deposit"
                           element={<Deposit></Deposit>}
-                    ></Route>*/}
+                        ></Route>
```

Comment:
Aren't we removing the deposit view to have it as a component?

Reply:
Yeah I plan to remove these eventually